src/main/java/edu/bu/metcs673/trackr/bankaccount/BankAccountService.java

```diff
@@ -16,50 +16,56 @@
 @Component
 public interface BankAccountService {
 
-    /**
-     * Creates a new bank account using the provided object, associate to the user
-     * making the request. Fields not provided by the user will be filled in with
-     * their default values.
-     *
-     * @param bankAccountInput
-     * @return BankAccount
-     */
-    public BankAccount createBankAccount(BankAccountDTO bankAccountInput, TrackrUser user);
+	/**
+	 * Creates a new bank account using the provided object, associate to the user
+	 * making the request. Fields not provided by the user will be filled in with
+	 * their default values.
+	 *
+	 * @param bankAccountInput
```

Comment:
You should also add parameter description in all doc comments.
` * @param bankAccountInput Bank account DTO`